### PR TITLE
add tor meetup event

### PR DESCRIPTION
**Pull request by francisco-core**

### CODE REVIEW

File: content/events/tor-meetup-brussels.en.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: "Brussels: Tor meetup"
+author: "privacylx"
+publishdate: "2019-10-30T11:00:00"
+date: "2019-11-08T18:00:00"
+image: "/img/tor-brussels-meetup.png"
+preview: true
+description: "Recovering our lost ground on digital privacy might seem like an overwhelming task, but in fact it is not as hard. In this workshop, we will explore and explain how to can help us in that as well as have a gathering of people interested in these topics"
+---
+
+![workshop image](/img/tor-brussels-meetup.png)
```
Comment (anadahz) [inline content/events/tor-meetup-brussels.en.md]: The image `tor-brussels-meetup.png` is missing, please add it.

File: content/events/tor-meetup-brussels.pt.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+publishdate: "2019-10-29T11:00:00"
+date: "2019-11-08T18:00:00"
+image: "/img/tor-brussels-meetup.png"
+preview: true
+description: "(EN) Recovering our lost ground on digital privacy might seem like an overwhelming taks, but in fact it is not as hard. In this workshop, we will explore and explain how to can help us in that as well as have a gathering of people interested in these topics"
+---
+
+*(apenas disponível em inglês)*
+
+![workshop image](/img/tor-brussels-meetup.png)
```
Comment (anadahz) [inline content/events/tor-meetup-brussels.pt.md]: The image `tor-brussels-meetup.png` is missing, please add it.

Comment (francisco-core) [reply content/events/tor-meetup-brussels.pt.md]: done

**Comment by francisco-core** [comment]
> Thank you for working on this.
> I added some fixes in [6923ea1](https://github.com/PrivacyLx/website/commit/6923ea10bb9b59be2f9af471e7a33c9cf4f1e32e) .
> The PR looks good but it's missing the image mentioned in the posts.

Thanks for spotting that! Ready to merge! (but only after the #18)

**Comment by anadahz** [comment]
Thank you, resolved the conflicts and merged.

**Pull request merged**